src/applications/find-va-forms/api/index.js

```diff
@@ -0,0 +1,13 @@
+// Relative imports.
+import mockForms from '../constants/example.json';
+import { normalizeFormsForTable } from '../helpers';
+
+// `fetchFormsApi` will need to be updated to make the request to api.va.gov once the endpoint is ready.
+// fetch(`https://api.va.gov/find-va-forms?q=${query}`);
+export const fetchFormsApi = () => {
+  // Normalize the forms data we get back from the API resopnse.
+  const normalizedForms = normalizeFormsForTable(mockForms?.data);
```

Comment:
I think this `normalizeFormsForTable` function makes sense to be called from the reducer.

Reply:
Actually I see now there's JSX, so maybe not 🤷‍♂